components/brave_wallet/browser/blockchain_registry_unittest.cc

```diff
@@ -542,6 +811,63 @@ TEST(BlockchainRegistryUnitTest, GetPrepopulatedNetworksKnownOnesArePreferred) {
   EXPECT_EQ(found_networks[0]->chain_name, "Ethereum Mainnet");
 }
 
+TEST(BlockchainRegistryUnitTest, GetTopDapps) {
+  base::test::TaskEnvironment task_environment;
+  auto* registry = BlockchainRegistry::GetInstance();
+  absl::optional<DappListMap> dapp_lists_map = ParseDappLists(dapp_lists_json);
+  ASSERT_TRUE(dapp_lists_map);
+  registry->UpdateDappList(std::move(*dapp_lists_map));
+
+  // Top Ethereum dapps
+  for (size_t i = 0; i < 2; ++i) {
```

Comment:
I don't see what this loop does. What is the point of calling `GetTopDapps` twice?

Reply:
I am copying the convention from this test https://github.com/brave/brave-core/blob/191f53e759f8b02aa8f34a605ae94f6aa3312b76/components/brave_wallet/browser/blockchain_registry_unittest.cc#L209-L210

I'll add the same comment for now, please tell me if I should remove the loop.